src/liballoc/tests/btree/map.rs

```diff
@@ -950,7 +960,9 @@ create_append_test!(test_append_144, 144);
 create_append_test!(test_append_145, 145);
 // Tests for several randomly chosen sizes.
 create_append_test!(test_append_170, 170);
+#[cfg(not(miri))] // Miri is too slow
 create_append_test!(test_append_181, 181);
+#[cfg(not(miri))] // Miri is too slow
```

Comment:
Hm, how bad is this really? The fact that the `cfg` already exists indicates, I think, that I already "tamed" these tests and was fine with how long they took with that... but the tests may have changed since then, and/or I may be more patient than you are. ;)

Reply:
Just a few seconds per test (though notably more for the 239), but these "randomly chosen sizes" just seem a total waste of time to me; they all add some leaves 2 levels down from root. Time that could be spent testing exotic workflows like appending keys that are smaller than the current minimum or to a multi-level tree (or to a tree with a shared root).

Yes, you must be more patient than I am.

Reply:
More accurately, 10 seconds for test_append_170, 14 seconds for test_append_239.